index.html

```diff
@@ -36,5 +36,151 @@ <h5 class="connect">LET'S CONNECT</h5>
+            </div>
+          </div>
+
+          <div class="water3">
+            <div class="project3"><img src="./img/Snapshoot-project3.png" alt="project3"></div>
+            <h2 class="tonic">Tonic</h2>
+            <ul class="info">
+              <li class="tonic-detail">CANOPY</li>
+              <li><img src="./img/Counter.png" alt="punto"></li>
+              <li class="tonic-detail2">Back End Dev</li>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+            </ul>
+            <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+                <ul class="buttons">
+                    <li><button class="int-button">html</button></li>
+                    <li><button class="int-button">css</button></li>
+                    <li><button class="int-button">JavaScript</button></li>
+                </ul>
+            </div>
+            <div><button class="see">See Project</button>
+            </div>
+
+          </div>
+
+          <div class="water4">
+            <div class="project4"><img src="./img/Snapshoot-project4.png" alt="project4"></div>
+            <h2 class="multi-post">Multi-Post Stories</h2>
+            <ul class="info">
+              <li class="tonic-detail">CANOPY</li>
+              <li><img src="./img/Counter.png" alt="punto"></li>
+              <li class="tonic-detail2">Back End Dev</li>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+            </ul>
+            <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+                <ul class="buttons">
+                    <li><button class="int-button">html</button></li>
+                    <li><button class="int-button">css</button></li>
+                    <li><button class="int-button">JavaScript</button></li>
+                </ul>
+            </div>
+            <div><button class="see">See Project</button></div>
+          </div>
+        </div>
+      </section>
+    </div>
+
+    <section>
+      <div class="garden2">
+        <div class="water5">
+          <h1 class="abaout-me">About <br>Myself</h1>
+          <p class="about">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+          <section>
+            <h5 class="connect">LET'S CONNECT</h5>
+            <ul class="social-media">
+                <li><a href="https://www.twitter.com/@osvaldoBarrio20"><img src="./img/Vector3.png" alt="twitter"></a></li>
+                <li><a href="https://www.linkedin.com/in/"><img src="./img/Linkedin.png" alt="Linkedin"></a></li>
```

Comment:
- [ ] Please always try to keep much attention to every details in the design to match it ,so that your project looks good and professional.so you should match different design aspects like font-colors,font-size, padding, margins ,etc.

your design:
![image](https://user-images.githubusercontent.com/90524466/185734251-0c36a3fb-b544-43a6-9ab5-8e6175798511.png)

Figma design:
![image](https://user-images.githubusercontent.com/90524466/185734333-f83086ba-5ef1-4a5c-bc56-2217e940b9a2.png)